src/common/state/table.cc

```diff
@@ -8,6 +8,8 @@ static const RetryInfo default_retry = {.num_retries = -1,
                                         .timeout = 10000,
                                         .fail_callback = NULL};
 
+static int64_t next_dummy_timer_id = 1000;
```

Comment:
Ok, I think I understand the problem. There are a ton of calls in `redis.cc` to

```c++
event_loop_remove_timer(db->loop, callback_data->timer_id);
```

so this will attempt to remove dummy timers from the event loop (which may correspond to real timers).